tests/apollo/util/bft.py

```diff
@@ -138,7 +139,9 @@ async def wrapper(*args, **kwargs):
                                         stop_replica_cmd=None,
                                         num_ro_replicas=num_ro_replicas)
                     with BftTestNetwork.new(config) as bft_network:
-                        bft_network.current_test = async_fn.__name__ + "_n=" + str(bft_config['n']) \
+                        storage_type = os.environ.get("STORAGE_TYPE")
+                        bft_network.current_test = async_fn.__name__ + "_" + storage_type \
```

Comment:
can't we use the TEST_NAME variable here?

Reply:
In case someone runs it locally and won't provide the storage name properly or won't provide it at all, extracting the storage type out of the TEST_NAME may produce strange log files names.

Reply:
AFAIK, you can't even run the tests locally without providing STORAGE_TYPE. Have you tried?

Reply:
You can run a single test with TEST_NAME equals any name, Therefore extracting the storage type from the test name will cause us problems.

Reply:
Can't you just say:
```python
bft_network.current_test = TEST_NAME
```

Reply:
OK, agreed.